apps/backend/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Lets not have a gitignore outside of an application repo. Move this to `apps/backend/{someting}/.gitignore`